File: JAVA/Car.java
class Car{
    private Integer id;
    private String license;
    private Account driver;
    private Integer passengers;

    public Car (String license, Account driver ) {
        this.license = license;
        this.driver = driver;
       
        
    }

    void printDataCar(){
        System.out.println("License: " + license + "Name Driver: " + driver.name + "Passengers" + passengers)
    }


    public Integer  getPassengers(){
        return passengers;
    }

    public void setPassengers(Integer passengers){
        if(passenger==4){
            this.passengers = passengers;
        }else{
            System.out.println("Necesitas asignar 4 puestos");
        }
    }

    

}

File: PHP/Car.php
<?php

require_once('Account.php');
class Car{
    public $id;
    public $license;
    public $driver;
    protected $passengers;

    public function __construct($license,$driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        echo "
            license: $this->license 
            Driver: {$this->driver->name} 
            Numero de pasajeros: $this->passengers
        
        
        ";
        
    }

    public function  getPassenger(){
        return $this->passengers;
    }

    public function setPassengers($passengers){ 
        if($passengers  == 4){
            $this->passengers = $passengers;
        }else{
            echo "Necesitas asignar 4 puestos";
        }
    }
}